src/battle-tooltips.ts

```diff
@@ -1484,6 +1484,9 @@ class BattleTooltips {
 		if (['psn', 'tox'].includes(pokemon.status) && move.category === 'Physical') {
 			value.abilityModify(1.5, "Toxic Boost");
 		}
+		if (this.battle.gen > 2 && serverPokemon.status === 'brn' && move.id !== 'facade' && move.category === 'Physical') {
+			if (!value.tryAbility("Guts")) value.modify(0.5, 'Burn');
```

Comment:
The reason this isn't here is because burn modifies the Attack stat, not the move's Base Power.

Reply:
Is that wrong since Gen 2?

Reply:
Apparently yes. Did not know. Oh well, I definitely massively prefer it this way, because it's much more obvious what's going on in the tooltip.

Reply:
Except this is more accurate; Burn cuts the damage of Body Press/Foul Play, which isn't immediately obvious with the way it's currently implemented